src/Tribe/Tickets.php

```diff
@@ -4029,7 +4029,9 @@ public function ticket_add( $post_id, $data ) {
 				$ticket->end_date = date( Tribe__Date_Utils::DBDATEFORMAT, strtotime( $end_datetime ) );
 			}
 
-			update_post_meta( $ticket->ID, '_type', $data['ticket_type'] ?? 'default' );
+			if ( $update ) {
+				update_post_meta( $ticket->ID, '_type', $data['ticket_type'] ?? 'default' );
+			}
```

Comment:
Is it still getting added while creating new tickets? We need to have this added always.

Reply:
It's always added - check a couple of lines below. I am just preventing trying to call it when ->ID is null